### PR TITLE
Update link on FAQ from the Spanish locale

### PR DESCRIPTION
**Pull request by itaquito**
Hello!

I noticed that the FAQ on Spanish references [a link](https://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/config/locales/es.yml#L3611) that doesn't work anymore. I updated this translation so it matches the [English one](https://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/config/locales/en.yml#L1826).

Thanks!

### CODE REVIEW

**Comment by gregorbg** [comment]
Thank you! I have noticed that on top of changing the link, you also changed the text around the link. This requires changing the `original_hash` in the YML file which can be done via Jonatan's `Internationalize` tool: https://github.com/thewca/worldcubeassociation.org/wiki/Translating-the-website#internationalize

Please update the YAML that you can download after changing the string in Internationalize, so the hashes stay in sync.
We will also need some form of proof that your Spanish translation is valid. This can be a confirmation by any Spanish Delegate or existing translators (currently Raúl Morales Hidalgo and Carlos Méndez García-Barroso).

**Comment by gregorbg** [comment]
Thanks for submitting the updated file so promptly! The hash should have changed, curious why it didn't. Let me investigate for a moment...

**Comment by itaquito** [comment]
Thanks!

I did the translation with the `Internationalize`  tool but the hash didn't change, this seems a little odd to me... Also, some other parts of the file changed, I'm not sure if this is fine.

I contacted the delegate of my country, Mexico, but she doesn't use GitHub. I'm not sure how she can confirm my translation. 

Thanks for your quick response

**Comment by gregorbg** [comment]
The automated removals are outdated strings which are not present in the English file anymore. It's fine, you can just leave this.

I think I know what happened. In short, human error. Some time ago, the translation changed from the CubingUSA link to our own resources link. In commit https://github.com/thewca/worldcubeassociation.org/commit/d2b4283e, somebody must have accidentally "approved" the old translation as being consistent with the new English translation.

TL;DR You're fine, sorry for making you go through the extra trouble with Internationalize. Normally, that's necessary :joy:

**Comment by gregorbg** [comment]
Regarding approval of your translation: The Delegate can either create a GitHub account (very useful and highly encouraged when they want to be involved in future discussions about Regulation changes or Software feature changes!) or send an email to software@worldcubeassociation.org using the link URL of this Pull Request as a reference.

**Comment by 123xARU** [comment]
Hello, I'm Areli Gordillo Mexican delegate. I'm not sure how to use GitHub but I can trust Itaquito Translations. He is the best!
